src/components/Products/Pagination.jsx

```diff
@@ -1,65 +1,69 @@
-						3
-					</div>
-					<div className="w-12 md:flex justify-center items-center hidden  cursor-pointer leading-5 transition duration-150 ease-in  rounded-full  ">
-						4
-					</div>
-					<div className="w-12 md:flex justify-center items-center hidden  cursor-pointer leading-5 transition duration-150 ease-in  rounded-full  ">
-						5
-					</div>
-					<div className="w-12 md:flex justify-center items-center hidden  cursor-pointer leading-5 transition duration-150 ease-in  rounded-full ">
-						6
-					</div>
-					<div className="w-12 md:flex justify-center items-center hidden  cursor-pointer leading-5 transition duration-150 ease-in  rounded-full  ">
-						...
-					</div>
-				</div>
-				<div className="h-12 w-12 ml-1 flex justify-center items-center rounded-full cursor-pointer">
-					<svg
-						xmlns="http://www.w3.org/2000/svg"
-						className="h-6 w-6"
-						fill="none"
-						viewBox="0 0 24 24"
-						stroke="currentColor"
-					>
-						<path
-							strokeLinecap="round"
-							strokeLinejoin="round"
-							strokeWidth={2}
-							d="M17 8l4 4m0 0l-4 4m4-4H3"
-						/>
-					</svg>
-				</div>
-			</div>
-		</div>
-	);
-};
+  const pageNumbers = [];
 
-export default Pagination;
+  for (let i = 1; i <= Math.ceil(totalPosts / postsPerPage); i++) {
+    pageNumbers.push(i);
+  }
+
+  return (
+    <div className='py-3 flex justify-center text-center'>
+      <nav className='block'>
+        <ul className='flex pl-0 rounded list-none flex-nowrap'>
+          <li>
+            <a
+              onClick={(e) => {
+                e.preventDefault();
+                paginateBack();
+              }}
+              href="#"
+              className={
+                currentPage===1
+                ?"bg-red-300 hover:bg-blue-200 relative inline-flex items-center px-4 py-2 border text-sm font-medium pointer-events-none"
+                :"bg-red-300 hover:bg-blue-200 relative inline-flex items-center px-4 py-2 border text-sm font-medium"
+              }
+
+            >Anterior</a>
+          </li>
+          <li>
+            {pageNumbers.map((number) => (
+              <a
+              	key={number}
+                onClick={(e) => {
+                  e.preventDefault();
+                  paginate(number);
+                }}
+                href='#'
+                className={
+                  currentPage === number
+                    ? "mx-0.5 bg-blue bg-red-300 hover:bg-blue-200 relative inline-flex items-center px-4 py-2 text-sm font-medium"
+                    : "mx-0.5 bg-white bg-red-100 text-gray-500 hover:bg-blue-200 relative inline-flex items-center px-4 py-2 text-sm font-medium"
+                }
+              >
+                {number}
+              </a>
+            ))}
+          </li>
+          <li>
+            <a
+              onClick={(e) => {
+                e.preventDefault();
+                paginateFront();
+              }}
```

Comment:
Please move this small block of code to a new function, OR just add `e.preventDefault()` to `paginateFront` to slim down JSX logic/code/volume of characters